test/core/binary.wast

```diff
@@ -350,6 +350,62 @@
   "zero flag expected"
 )
 
+;; Local count can be 2^32.
+(module binary
```

Comment:
WABT currently fails this test, see https://github.com/WebAssembly/wabt/issues/1593